xarray/core/variable.py

```diff
@@ -1579,7 +1579,7 @@ def _unstack_once(
 
         # Potentially we could replace `len(other_dims)` with just `-1`
         other_dims = [d for d in self.dims if d != dim]
-        new_shape = list(reordered.shape[: len(other_dims)]) + new_dim_sizes
+        new_shape = tuple(list(reordered.shape[: len(other_dims)]) + new_dim_sizes)
```

Comment:
probably not worth a new PR, but if `shape` is always a tuple this could be simplified to
```suggestion
        new_shape = reordered.shape[: len(other_dims)] + tuple(new_dim_sizes)
```